components/home/about/About.tsx

```diff
@@ -0,0 +1,124 @@
+import { Col, Container, Row } from "react-bootstrap";
+import { DatabaseNodes } from "./DatabaseNodes";
+import { ToolNodes } from "./ToolNodes";
+import styles from "./About.module.scss";
+import Highlight, { defaultProps } from "prism-react-renderer";
+import { WigglyLine } from "./WigglyLine";
+import vsLight from "prism-react-renderer/themes/vsLight";
+
+const code = `
```

Comment:
What would you think about this code block being in dark mode (dark background, light text)? I think the contrast would make things a bit easier to read.

Reply:
![Screen Shot 2021-12-15 at 4 42 38 PM](https://user-images.githubusercontent.com/4967694/146269444-aaee3eb0-1ebb-4b0d-ad30-7f25933e528c.png)
?